src/index.js

```diff
@@ -1,37 +1,57 @@
 import 'bootstrap/dist/css/bootstrap.css'
 
 import { Container, Navbar, NavbarBrand } from 'reactstrap'
-import { css } from 'emotion'
+import { css, cx } from 'react-emotion'
 import { faCompass } from '@fortawesome/free-solid-svg-icons'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+
 import React from 'react'
 import ReactDOM from 'react-dom'
 
-import { GRAY_900 } from './assets/colors'
+import { GRAY_900, BLUE_GRAY_900 } from './assets/colors'
 import FeedbackButton from './components/feedback-button'
 import MortgageInvestmentCompare from './components/mortgage-investment-compare/index'
 
-function App () {
-  // TODO 018.12.09: Include an intro paragraph
-  return (
-    <div
-      className={css`
-        color: ${GRAY_900};
-        font-family: 'Noto Sans', sans-serif;
-      `}
-    >
-      <Navbar className='border-bottom sticky-top' color='white' light>
-        <NavbarBrand href='/'>
-          <FontAwesomeIcon icon={faCompass} className='mr-2 text-success' />
-          MoneyCompass
-        </NavbarBrand>
-      </Navbar>
-      <Container fluid className='pt-4 px-5'>
-        <MortgageInvestmentCompare />
-        <FeedbackButton />
-      </Container>
-    </div>
-  )
+const footer = cx(
+  css(`background-color: ${BLUE_GRAY_900};`),
+  'text-white',
+  'd-flex',
+  'flex-column',
+  'align-items-center',
+  'mt-5',
+  'p-4'
+)
+
+class App extends React.Component {
+  render () {
+    // TODO 018.12.09: Include an intro paragraph
+    return (
+      <div
+        className={css`
+          color: ${GRAY_900};
+          font-family: 'Noto Sans', sans-serif;
+        `}
+      >
+        <Navbar className='border-bottom sticky-top' color='white' light>
+          <NavbarBrand href='/'>
+            <FontAwesomeIcon icon={faCompass} className='mr-2 text-success' />
+            MoneyCompass
+          </NavbarBrand>
+        </Navbar>
+        <Container fluid className='pt-4 px-5'>
+          <MortgageInvestmentCompare />
+          <FeedbackButton />
+        </Container>
+        <footer className={footer}>
+          <p>Copyright © 2019 Money Compass. All rights reserved</p>
+          <p>
+            Say hi!{' '}
+            <a href='mailto:hello@moneycompass.org'>hello@moneycompass.org</a>
```

Comment:
We probably need to update standard link colors to match the light blue that our feedback button uses

Reply:
think we should go back to using injectGlobal? and move the font styling there too?

Reply:
i don't see why the icon's light blue should be the same blue as the link's blue though. They are different things no?

Reply:
Color-wise, they are meant to be more similar than different. That color should indicate to users that the thing is clickable

The color can probably be set w/ the main page's css class

Reply:
ok. So should I create a class with the following?

a {
 color: ${LIGHT_BLUE}
}

and then add that to the classname of the page's div where the font family currently lives?

Do you not think it would be cleaner to just put everything inside `injectGlobal`?

Reply:
both works!